admin_guide/overcommit.adoc

```diff
@@ -45,67 +45,42 @@ lowest quality of service.
 [[tune-buffer-chunk-limit]]
 === Tune Buffer Chunk Limit
 
-If Fluentd logger is unable to keep up with a high number of logs, you will need
-to increase the compute resource values.
+If Fluentd logger is unable to keep up with a high number of logs, it will need
+to switch to file buffering to reduce memory usage and prevent data loss.
 
```

Comment:
Does the fluentd logger make this switch on its own? Using the values below that the user configured?